mycli/main.py

```diff
@@ -428,10 +431,12 @@ def prompt_tokens(cli):
 
                 try:
                     logger.debug('sql: %r', document.text)
-                    if self.logfile:
+                    if self.logfile is not False:
                         self.logfile.write('\n# %s\n' % datetime.now())
                         self.logfile.write(document.text)
                         self.logfile.write('\n')
+                    else:
+                        self.output("Error: Unable to load the audit log file.", err=True, fg='red')
```

Comment:
At this point we aren't loading the file. Maybe a more helpful error message might be `...write to the audit log file.`